invenio_rdm_records/records/dumpers.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""DataCite-based data model for Invenio."""
+
+import calendar
+from datetime import date
+
+from edtf import parse_edtf
+from invenio_records.dumpers import ElasticsearchDumperExt
+
+
+class EDTFDumperExt(ElasticsearchDumperExt):
```

Comment:
The name I think is fine, but perhaps instead of hardcoding the field name, we could provide it directly on instantiation:

``EDTFDumperExt('metadata.publication_date')``

Reply:
Implemented:

- It requires a dot separated patht to the field: https://github.com/inveniosoftware/invenio-rdm-records/pull/194/files#diff-3062ab2ea652c3ad7e17249676a44495cdc1eaf6f3c7b661c071c94dfccf06aaR34
- Then _dump and _load do the work recursively